### PR TITLE
feat: configuring number of replicas

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
Configuring number of replicas

## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/22067

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
+:_content-type: PROCEDURE
+:description: Configuring a number of replicas
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:2]:
```suggestion
:description: Configuring the number of replicas
```

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+:keywords: administration-guide, configuring-number-of-replicas
+:navtitle: Configuring a number of replicas {prod-cli}
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:4]:
```suggestion
:navtitle: Configuring the number of replicas {prod-cli}
```

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+:page-aliases:
+
+[id="configuring-number-of-replicas"]
+= Configuring a number of replicas
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc]:
```suggestion
= Configuring the number of replicas
```

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+To configure the number of replicas for {prod-short} operands using {kubernetes} `HorizontalPodAutoscaler`
+(HPA), you can define an `HPA` resource for deployment. The `HPA` dynamically adjusts the number
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:11]:
```suggestion
(HPA), define the `HPA` resource for deployment. The `HPA` dynamically adjusts the number
```

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+of replicas based on specified metrics.
+
+.Procedure
+
+. Create an `HPA` resource for a deployment, specifying the target metrics and desired replica count.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:16]:
```suggestion
. Create an `HPA` resource for the deployment, specifying the target metrics and desired replica count.
```

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  ...
+----
+<1> The `__<deployment_name>__` corresponds to the one following deployments:
+* `{prod-deployment}`
+* `che-gateway`
+* `{prod-deployment}-dashboard`
+* `plugin-registry`
+* `devfile-registry`
+
+.Create a `HorizontalPodAutoscaler` for {prod-deployment} deployment:
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:39]:
```suggestion
. Create a `HorizontalPodAutoscaler` for {prod-deployment} deployment:
```

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+  maxReplicas: 5
+  metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 75
+----
+In this example, the HPA is targeting the Deployment named {prod-deployment}, with a minimum of 2 replicas, a maximum of 5 replicas
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:63]:
```suggestion
In this example, the HPA is targeting the Deployment named {prod-deployment}, with the minimum of 2 replicas, the maximum of 5 replicas,
```

**Review by deerskindoll — COMMENTED** [review]
small nits + a Vale fix

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc (hunk 6 above)
Comment (tolusha) [reply modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:39]: It is a example title. `_` is not allowed.

Comment (deerskindoll) [reply modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc:39]: I thought it's the next step in the instructions. sorry, my bad.

File: modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc (hunk 3 above)
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc]:
```suggestion
= Configuring the number of replicas
```

**Review by deerskindoll — COMMENTED** [review]
additional Vale error found

**Comment by tolusha** [comment]
@deerskindoll 
Can I merge?

**Review by deerskindoll — APPROVED** [review]
lgtm

**Comment by deerskindoll** [comment]
> @deerskindoll Can I merge?

@tolusha go for it

**Pull request merged**